### PR TITLE
[Merged by Bors] - chore: forward-port mathlib#19204

### PR DESCRIPTION
**Pull request by xroblot**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by riccardobrasca — APPROVED** [review]
Just check that the names in the various comments are the Lean4 names, otherwise LGTM, thanks!

bors d+

File: Mathlib/Algebra/Invertible.lean [hunk 1]
```diff
@@ -358,6 +373,56 @@ instance (priority := 100) Invertible.ne_zero [MulZeroOneClass α] [Nontrivial 
...
+  toFun _ := invertibleMul a b
+  invFun _ := invertibleOfMulInvertible _ b
+  left_inv _ := Subsingleton.elim _ _
+  right_inv _ := Subsingleton.elim _ _
+#align invertible.mul_right Invertible.mulRight
+
+end Monoid
+
+
+
```
Comment (eric-wieser) [inline Mathlib/Algebra/Invertible.lean:425]:
```suggestion

```

File: Mathlib/Algebra/Invertible.lean [hunk 2]
```diff
@@ -290,12 +297,20 @@ def invertibleMul [Monoid α] (a b : α) [Invertible a] [Invertible b] : Inverti
   ⟨⅟ b * ⅟ a, by simp [← mul_assoc], by simp [← mul_assoc]⟩
 #align invertible_mul invertibleMul
 
+
```
Comment (eric-wieser) [inline Mathlib/Algebra/Invertible.lean:300]: Stray newlines

**Comment by eric-wieser** [comment]
bors merge

Thanks!

**Pull request closed without merging**